frontends/web/src/routes/new-settings/components/tabs.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReactNode } from 'react';
+import { Main, Header } from '../../../components/layout';
+import { View, ViewContent } from '../../../components/view/view';
+import { MobileHeader } from './mobileheader';
+import i18n from '../../../i18n/i18n';
+import { NavLink } from 'react-router-dom';
+import styles from './tabs.module.css';
+import { route } from '../../../utils/route';
+import { SettingsItem } from './settingsItem/settingsItem';
+import { RightChevron } from '../../../components/icon';
+
+type TTabsProps = {
+  settingsTabsDetail: TTab[];
+}
+
+type TWithSettingsTabsProps = {
+  children: ReactNode
+}
+
+export type TTab = {
+  url: string;
+  name: string;
+}
+
+const settingsTabsDetail: TTab[] = [
+  { url: '/new-settings/appearance', name: i18n.t('settings.appearance') },
+  { url: '/new-settings/manage-accounts', name: 'Manage accounts' },
+  { url: '/new-settings/device-settings', name: 'Device settings' },
```

Comment:
keep in mind, 'Device settings' should only be shown when there is a device plugged in and unlocked. 

Also the logic currently would in theory support multiple deivces. Please take the same logic

https://github.com/digitalbitbox/bitbox-wallet-app/blob/697699f6f874ef0ecafb33ef3ed33629faf0a6ec/frontends/web/src/components/sidebar/sidebar.tsx#L212-L225

Reply:
Good catch, a missing piece of info for me here. Thanks!